go/parquet/pqarrow/encode_arrow_test.go

```diff
@@ -1526,6 +1528,7 @@ func TestWriteTableMemoryAllocation(t *testing.T) {
 			arrow.Field{Name: "i64", Type: arrow.PrimitiveTypes.Int64, Nullable: true},
 			arrow.Field{Name: "f64", Type: arrow.PrimitiveTypes.Float64, Nullable: true})},
 		{Name: "arr_i64", Type: arrow.ListOf(arrow.PrimitiveTypes.Int64)},
+		{Name: "uuid", Type: types.NewUUIDType(), Nullable: true},
```

Comment:
please add this to one of the round trip tests (this test doesn't perform a read) so that we have full testing for reading *and* writing the extension type along with confirmation that the resulting Array we read matches what we wrote.

Reply:
Added as of [these commits](https://github.com/apache/arrow/pull/34631/files/517ace50760cad5749a3c1e927372c90e6fd5b85..05295d3a8ce9d6f619163d0217bdeccd37aaab86)